lib/Sabberworm/CSS/Parser.php

```diff
@@ -480,6 +483,16 @@ private function parseNumericValue($bForColor = false) {
 		return new Size(floatval($sSize), $sUnit, $bForColor, $this->iLineNo);
 	}
 
+	private function parseLineNameValue() {
+		$this->consume('[');
+		$sName = '';
```

Comment:
Use `consumeWhitespace` after `[` and before `]`.

Reply:
Should okay now. Please check.